### DISCUSSION TITLE
Included plugins?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [asmcss/assembler#23] [Ideas]
Wouldn't it be cool to have included plugins, these could be in a new `plugins` folder and would be in JS format. There could also be a plugins.json somewhere with plugin descriptions, names and filenames. This would allow the community to create plugins easily, and possibly have them incorporated into the main project! It would also keep plugins organized as the community continues to grow, and more plugins are developed

**Comment by Awilum** [comment]
Same question from me https://github.com/asmcss/assembler/issues/11

**Comment by Explosion-Scratch** [comment]
> Same question from me #11

I saw yours, it seems like your issue is asking about any existing plugins and how to create plugins, while mine is more of a suggestion for included plugins with the library, that could be accessed via the CDNs e.g. a /plugins folder in dist which would have stuff like `buttons-plugin.min.css` with mixins and stuff.

Feel free to correct me if I'm wrong! 😄

**Comment by msarca** [comment]
Honestly, I would rather like to see that people build UI components instead of plugins.
